kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/MergeQueriesTest.java

```diff
@@ -123,19 +123,20 @@ public void testUpsertQueryWithPartitionTime() {
             + "GROUP BY key.k1, key.k2.nested_k1.doubly_nested_k, key.k2.nested_k2)) "
           + "ON `" + DESTINATION_TABLE.getTable() + "`." + KEY + "=src.key "
           + "WHEN MATCHED "
-            + "THEN UPDATE SET f1=src.value.f1, f2=src.value.f2, f3=src.value.f3, f4=src.value.f4 "
+            + "THEN UPDATE SET `f1`=src.value.f1, `f2`=src.value.f2, `f3`=src.value.f3, `f4`=src.value.f4 "
           + "WHEN NOT MATCHED "
-            + "THEN INSERT ("
-              + KEY + ", "
+            + "THEN INSERT (`"
+              + KEY + "`, "
               + "_PARTITIONTIME, "
-              + "f1, f2, f3, f4) "
+              + "`f1`, `f2`, `f3`, `f4`) "
             + "VALUES ("
               + "src.key, "
               + "CAST(CAST(DATE(src.partitionTime) AS DATE) AS TIMESTAMP), "
               + "src.value.f1, src.value.f2, src.value.f3, src.value.f4"
             + ");";
     String actualQuery = mergeQueries(true, true, false)
         .mergeFlushQuery(INTERMEDIATE_TABLE, DESTINATION_TABLE, BATCH_NUMBER);
+    System.out.println(actualQuery);
```

Comment:
Can we remove this part? The failure messages from `assertEquals` should include both the expected query and the actual query if there are any test failures.

Reply:
Sure, done!